src/app/resources/view-resources/resources-view.scss

```diff
@@ -36,4 +36,10 @@
       max-height: 60vh;
     }
   }
+
+  button{

```

Comment:
button styling is making resource view button to look nice. button on other pages are still breaking. Can we have the general style for title-header may be using some common style.